MicaForEveryone.UI/Strings/ko/Resources.resw

```diff
@@ -121,31 +121,31 @@
     <value>앱 정보</value>
   </data>
   <data name="AcrylicBackdropDescription.Text" xml:space="preserve">
-    <value>아크릭은 모든 앱에 윈도우 10 스타일 이펙트를 적용합니다.</value>
+    <value>아크릭은 프로그램 창에 윈도우 10 아크릭 효과를 적용합니다.</value>
```

Comment:
How about `아크릴` instead of '`아크릭`?
Microsoft [spells it `아크릴`](https://learn.microsoft.com/ko-kr/windows/apps/design/style/acrylic).

Reply:
They do? Guess i need to change some parts again.

and of course my keyboard is broken again

Thanks for looking for the translation errors instead of me